plugins/code-mirror/script.js

```diff
@@ -4,7 +4,16 @@
  * @author Bhsd <https://github.com/bhsd-harry>
  */
 ;(async () => {
-  const MODE_LIST = {
+  const codemirrorInstalled = mw.loader.getState('ext.CodeMirror')
+  const settings = JSON.parse(localStorage.getItem('InPageEditMwConfig')) || {}
+  const serverName = mw.config.get('wgServerName')
```

Comment:
We'd better use a combination of wgServerName and wgArticlePath. Because wiki in different languages on Fandom are using same domain.

Reply:
I used wgScriptPath instead.